stackit/internal/services/argus/instance/resource_test.go

```diff
@@ -884,6 +948,16 @@ func TestToUpdateAlertConfigPayload(t *testing.T) {
 				Route: types.ObjectValueMust(routeTypes, map[string]attr.Value{
 					"receiver": types.StringValue("example-receiver"),
 				}),
+				GlobalConfiguration: types.ObjectValueMust(globalConfigurationTypes, map[string]attr.Value{
+					"opsgenie_api_key":   types.StringValue("key"),
+					"opsgenie_api_url":   types.StringValue("ops.example.com"),
+					"resolve_timeout":    types.StringValue("1m"),
+					"smtp_auth_identity": types.StringValue("identity"),
+					"smtp_auth_username": types.StringValue("username"),
+					"smtp_auth_password": types.StringValue("password"),
+					"smtp_from":          types.StringValue("me@example.com"),
+					"smtp_smart_host":    types.StringValue("smtp.example.com:25"),
+				}),
```

Comment:
Maybe it's worth making this a fixture too? It's the same as the one from the test above, right?